src/server/pps/server/api_server.go

```diff
@@ -1567,27 +1506,6 @@ func branchProvenance(input *pps.Input) []*pfs.Branch {
 	return result
 }
 
-// writePipelineInfo is a helper for StartPipeline and StopPipeline that writes
-// out an updated pipeline info (without changing the version).  It is an error
-// if the pipeline has been updated and is no longer the expected version.
-func (a *apiServer) writePipelineInfo(ctx context.Context, pipelineInfo *pps.PipelineInfo) (*pfs.Commit, error) {
```

Comment:
We no longer write out new spec commits when starting/stopping pipelines, so this function becomes unused.